### DISCUSSION TITLE
Adding images in code documentation

### DISCUSSION THREAD

**Discussion by astrochun** [mkdocstrings/mkdocstrings#456] [Q&A] [answered]
Hi, I'm using the [following recipe](https://mkdocstrings.github.io/recipes/#automatic-code-reference-pages) to have automatic documentation for my codes, modules, etc.

I'm trying to embed an image to illustrate the outcome in one of the `__init__.py` files with a top-level docstrings. I've added the files in a folder called `docs/assets`; however, when trying to include it via any of the following, I'm getting a 404 error:

```
![](../../docs/assets/example.png)
```

```
![](/docs/assets/example.png)
```

I either get:
```
WARNING  -  [12:30:07] "GET /project_name/docs/assets/example.png HTTP/1.1" code 404
WARNING  -  [12:29:15] "GET /project_name/reference/project_name/docs/assets/example.png HTTP/1.1" code 404
```

I think it's straightforward to add such a file in index.md but to do so with autodoc, it's not possible. It seems like I need to find a way within `mkdocs` to add such assets.

**Comment by pawamoy** [comment, marked as answer]
Images should definitely be added using relative links. You just need to think about the following things:

- in what path will my docstring be rendered?
- what's the relative path to the image from there?

The docs folder most certainly shouldn't be used in the relative link as it's the root of the docs file (usually).

So given your input I'd bet on:

```
![](../../assets/orcid_example.png)
```

**Reply by astrochun** [reply]
Thanks for the quick response! Leaving that extra `docs` was the mistake.
